### ISSUE TITLE
Image shows black outline in Windows

### ISSUE THREAD

**Issue by rlam1** [tdicola/creepyportrait#2]
The same issue found in the Raspberry Pi with the memory splits also happens in windows when Normal Mapping is enabled, however that solution is not possible this time :)

![image](https://f.cloud.github.com/assets/1676233/1443205/1a82f052-41db-11e3-8f12-47a674839de8.png)

**Comment by tdicola** [comment]
Ah interesting, it looks like something is going wrong in the shader.  Do you know what graphics hardware you're using?

**Comment by rlam1** [comment]
Yes, it is a nvidia gt 610 with current drivers. 2gb ram. With support for directX11, running on an Intel core i3 second gen. Maybe this card does not support that shader, as it is a basic one.
